code/game/objects/items/tanks/watertank.dm

```diff
@@ -171,6 +171,37 @@
 /obj/item/reagent_containers/spray/mister/janitor/mode_change_message(mob/user)
 	to_chat(user, span_notice("You [amount_per_transfer_from_this == 10 ? "remove" : "fix"] the nozzle. You'll now use [amount_per_transfer_from_this] units per spray."))
 
+//Security tank
+/obj/item/watertank/security
+	name = "ANTI-TIDER-2500"
+	desc = "The ultimate crowd-control weapon ; this tool allows the user to quickly and efficiently pacify groups of hostile targets."
+	icon = 'icons/obj/hydroponics/equipment.dmi'
+	icon_state = "pepperbackpacksec"
+	inhand_icon_state = "pepperbackpacksec"
+	custom_price = PAYCHECK_HARD * 2
+
+/obj/item/watertank/security/Initialize()
+	. = ..()
+	reagents.add_reagent(/datum/reagent/consumable/condensedcapsaicin, 500)
+
+/obj/item/reagent_containers/spray/mister/security
+	name = "security spray nozzle"
+	desc = "A pacifying spray nozzle attached to a pepperspray tank, designed to silence perps."
+	icon = 'icons/obj/hydroponics/equipment.dmi'
+	icon_state = "mistersecjani"
+	inhand_icon_state = "mistersecjani"
+	lefthand_file = 'icons/mob/inhands/equipment/mister_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/mister_righthand.dmi'
+	amount_per_transfer_from_this = 5
+	possible_transfer_amounts = list(5,10)
+	current_range = 6
+
+/obj/item/watertank/security/make_noz()
+	return new /obj/item/reagent_containers/spray/mister/security(src)
+
+/obj/item/reagent_containers/spray/mister/security/mode_change_message(mob/user)
+	to_chat(user, span_notice("You [amount_per_transfer_from_this == 10 ? "remove" : "fix"] the nozzle. You'll now use [amount_per_transfer_from_this] units per spray."))
```

Comment:
Should this be 'affix' rather than 'fix'? I know it'd be inconsistent with other things, but it'd make more sense. Maybe update it for all the sprayers?